specifyweb/frontend/js_src/lib/components/DataModel/businessRules.ts

```diff
@@ -172,21 +171,6 @@ export class BusinessRuleManager<SCHEMA extends AnySchema> {
         );
       }
     );
-
-    Promise.all(results).then((results) => {
-      results
-        .map((result: BusinessRuleResult<SCHEMA>) => result['localDuplicates'])
-        .flat()
-        .filter((result) => result !== undefined)
-        .forEach((duplicate: SpecifyResource<SCHEMA> | undefined) => {
-          if (duplicate === undefined) return;
-          const event = duplicate.cid + ':' + (fieldName as string);
-          if (!this.watchers[event]) {
-            this.watchers[event] = () =>
-              duplicate.on('change remove', () => this.checkField(fieldName));
-          }
-        });
-    });
```

Comment:
a lot of (potentially problematic) code got removed here. just double checking that that was intentional

Reply:
Yes this was intentional. 
But in hindsight I realize that removing this created a bug. 

Fixed in https://github.com/specify/specify7/pull/3976/files/55cdae94cd69a21d7326c92060c04d8e7e9f946e..b94e85ae2ae919174ab854ec1a27e5964d1d2dc5